### DISCUSSION TITLE
Huion HS611 not detected by driver

### DISCUSSION THREAD

**Discussion by m4mux** [OpenTabletDriver/OpenTabletDriver#2714] [Q&A] [answered]
I've tried the advice online but none seem to work, uninstall Razer Synapse, Huion Driver and clear remainders left ect.
Nothing gets my tablet detected by the driver.

I've attached my diagnostic file below, and also I'm planning to use the tablet to play Valorant if that means anything.
[20mux.txt](https://github.com/OpenTabletDriver/OpenTabletDriver/files/11303756/20mux.txt)

**Comment by jamesbt365** [comment, marked as answer]
This has been resolved in the latest release of OpenTabletDriver.

**Reply by m4mux** [reply]
Haha, thanks for replying. I've sold my HS611 now lol
